M0narc/fast_api
# no-story - put endpoint for todos

## Description

update endpoint for todos


## Testing Instructions
grab any todos id and use it in the update endpoint, modify it and then check if it has been updated.


## Screenshots


![imagen](https://github.com/M0narc/fast_api/assets/46531321/eeafd82f-c2ef-4609-89c2-7b06bf068158)
